GeneralsMD/Code/GameEngine/Source/Common/System/BuildAssistant.cpp

```diff
@@ -874,18 +876,27 @@ LegalBuildCode BuildAssistant::isLocationClearOfObjects( const Coord3D *worldPos
 			/* Check for overlap of my exit rectangle to his geom info. */
 			if (checkMyExit && ThePartitionManager->geomCollidesWithGeom(them->getPosition(), hisBounds, them->getOrientation(),
 				&myExitPos, myGeom, angle)) {
+				if (them->getDrawable() && them->getDrawable()->getFullyObscuredByShroud())
```

Comment:
Edits to Zero Hour are different to Generals.

Reply:
Hmm, you're right. Not sure why I did that.